pkgs/racket-doc/scribblings/reference/syntax.scrbl

```diff
@@ -2787,13 +2787,14 @@ For backward compatibility only; equivalent to @racket[syntax-local-introduce].
 @defform*[[(begin form ...)
            (begin expr ...+)]]{
 
-The first form applies when @racket[begin] appears at the top level,
-at module level, or in an internal-definition position (before any
-expression in the internal-definition sequence). In that case, the
+The first form applies when @racket[begin] appears at the @tech{top-level context},
+at @tech{module context}, or in an @tech{internal-definition context}. In that case, the
 @racket[begin] form is equivalent to splicing the @racket[form]s into
 the enclosing context.
+The @tech{internal-definition context} itself is said to have an
+@deftech{implicit begin} of this first form.
```

Comment:
"of the first form" is correct, but how about "The internal-definition context itself is treated as a begin containing definitions" or something more like that?